### ISSUE TITLE
Accuracy didnt increase 

### ISSUE THREAD

**Issue by Timmmeyyy** [brakid/BundesligaPrediction#1]
Hi, i have a probleme that my accuracy didnt increase, it is always 0,66 after 1400 epochs. Do you have any idea whtat can i check first. I think my input data and labels are right. Greetz Tim

**Comment by brakid** [comment]
Hi,

maybe you need to tweak a bit the hyperparameters of the model such as the _layer-count and size_: https://github.com/brakid/BundesligaPrediction/blob/2b647f2b8a585b8dcbcfb91ff21434cf48b68ee0/prediction.py#L178 and following lines. You might also want to check different activation functions such as _tanh_ or _softrelu_ in the layers: https://beta.mxnet.io/api/ndarray/_autogen/mxnet.ndarray.Activation.html

In general, if the model fails to match your data, this is a sign that the model is not powerful enough to capture the subtleties in your dataset, _adding additional layers_ could be helpful as well.

Another option could be that the optimizer is not calibrated correctly for your data. The _learning rate_ might be too high or low: https://github.com/brakid/BundesligaPrediction/blob/2b647f2b8a585b8dcbcfb91ff21434cf48b68ee0/prediction.py#L191 

There are various resources on hyperparameter tuning for Neural networks, such as: https://towardsdatascience.com/hyper-parameter-tuning-techniques-in-deep-learning-4dad592c63c8.

Hopefully this helps you increase the accuracy and performance of your neural network.
